<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginForm</class>
 <widget class="QMainWindow" name="loginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#statusbar
{
	color: white;
	border-radius: 15px;
}
#centralwidget
{
	border-radius: 15px;
}
#loginForm
{

background:#2D2D2D;
border-radius: 15px;
	
}

#frame
{
	background:#2D2D2D;
	border-radius: 15px;
}



#toolButton
{
	background: red;
	border-radius:30px;
}

#label_5
{
	color: white;
background: transparent;
	font-family: century gothic;
	font-size: 24px
}
#label_6
{
	color: white;
background: transparent;
	font-family: century gothic;
	font-size: 24px
}




</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>521</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_login">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>87</y>
         <width>391</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
background: transparent;
border: none;
color: #717072;
border-bottom: 1px solid  #717072;
	font-family: century gothic;
	font-size: 24px
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>149</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
background: transparent;
border: none;
color: #717072;
border-bottom: 1px solid  #717072;
	font-family: century gothic;
	font-size: 24px
}

</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_login">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
#pushButton_login
{
	color: white;
	border-radius: 15px;
	background: red;
	font-family: century gothic;
	font-size: 24px

}

#pushButton_login:hover
{
	color: #333;
	border-radius: 15px;
	background: #49ebff;
	font-family: century gothic;
	font-size: 24px

}

</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>37</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>LOGIN HERE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="minimizeButton">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>0</y>
         <width>30</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
#minimizeButton
{
	border:none;
color:white;
}

#minimizeButton:hover
{
	border:none;
color:#EB0000;
}</string>
       </property>
       <property name="text">
        <string>−</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#closeButton
{
	border:none;
color:white;
}

#closeButton:hover
{
	border:none;
color:#EB0000;
}</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>20</y>
        <width>411</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; color:#ffffff;&quot;&gt;Non connected&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#closeButton_2
{
	border:none;
color:white;
}

#closeButton_2:hover
{
	border:none;
color:#EB0000;
}</string>
      </property>
      <property name="text">
       <string>×</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
